src/Pyz/Zed/Console/ConsoleDependencyProvider.php

```diff
@@ -458,10 +458,6 @@ protected function getConsoleCommands(Container $container): array
             $commands[] = new RouterDebugBackofficeConsole();
             $commands[] = new RouterDebugBackendGatewayConsole();
             $commands[] = new RouterDebugBackendApiConsole();
-
-            $commands[] = new SprykRunConsole();
-            $commands[] = new SprykDumpConsole();
-            $commands[] = new SprykBuildConsole();
```

Comment:
Wrong  direction?
You should add the use Statements again, not remove functionality

Reply:
No, I've discussed it with @stereomon, this way seems to be correct.

> Das wäre also klar?
![image](https://user-images.githubusercontent.com/4996022/160787651-b90876a4-a44b-4d80-ba7d-9c378b2d1602.png)
[09:50](https://spryker.slack.com/archives/D01JWNCKTTK/p1648626627674779)
[René Klatt](https://app.slack.com/team/U03DJTYAA) Ja die müssen weg.

Reply:
Are they?
But how does spryk work in suite then?
With what commands?

Reply:
I was wrong. This is suite and it uses public released version of spryks which is fine. Use statements and commands should stay here.

Reply:
I rather think suite nonsplit was wrong where it was placed in the wrong part